docs/user-guide.md

```diff
@@ -135,6 +135,36 @@ sshKey: <redacted>
 userCorePass: <redacted>
 ```
 
+### Add custom manifests (Optional)
+* Note that any manifest added here will apply to **any** of the clusters installed using this image.
+* Find more details and additional examples in OpenShift documentation: [Customizing nodes](https://docs.openshift.com/container-platform/4.13/installing/install_config/installing-customizing.html).
+
```

Comment:
Also some example for 'Post-installation configuration': https://docs.openshift.com/container-platform/4.13/post_installation_configuration/machine-configuration-tasks.html#using-machineconfigs-to-change-machines

Reply:
Done